test/unit/spec/v1/PrimaryAuth_spec.js

```diff
@@ -204,9 +224,19 @@ function setupSocial(settings) {
       settings
     )
   ).then(function(test) {
+    if (options.mockPopup) {
+      spyOn(test.ac.token, 'getWithPopup').and.callFake(function() {
```

Comment:
nit: when we are touching `spyOn` let's change to `jest.spyOn(...).mockImplementation(callback)` when possible. It may break some assertions since the API of the returned mocks are a bit different. Some of the `expect` matchers for mocks in jest make things easier though.